rustls-platform-verifier/src/tests/verification_real_world/mod.rs

```diff
@@ -118,6 +118,35 @@ macro_rules! no_error {
 fn real_world_test<E: std::error::Error>(test_case: &TestCase<E>) {
     log::info!("verifying {:?}", test_case.expected_result);
 
+    // On BSD systems openssl-probe fails to find the system CA bundle,
+    // so we must provide extra roots from webpki-roots.
+    #[cfg(all(
```

Comment:
Ditto on these two `cfg` blocks that were added as well.

Reply:
Fixed here too.